quinn-proto/src/connection/pacing.rs

```diff
@@ -0,0 +1,73 @@
+//! Pacing of packet transmissions.
```

Comment:
Let's squash this commit into the other one.

Reply:
Done